pallets/gear/src/benchmarking/mod.rs

```diff
@@ -422,31 +421,44 @@ benchmarks! {
     }
 
     send_message {
-        let p in 0 .. MAX_PAYLOAD_LEN;
+        let v in 0 .. 1; // value is zero or no
+        let p in 0 .. 1; // dest is program or regular account
         let caller = benchmarking::account("caller", 0, 0);
```

Comment:
If we add here following line: `log::debug!("v: {v} p: {p}");`
then following commands
```shell
$ cargo build -p gear-node --release --features runtime-benchmarks
$ RUST_LOG=pallet_gear=debug ./target/debug/gear-node benchmark pallet --chain=dev \
--steps=1 --repeat=1 --pallet=pallet_gear --extrinsic="send_message" --execution=wasm \
--wasm-execution=compiled --heap-pages=4096
```
show that there is no case when v = 0 p = 0